app/src/main/java/ch/epfl/sdp/appart/scrolling/ScrollingViewModel.java

```diff
@@ -20,6 +20,7 @@
 public class ScrollingViewModel extends ViewModel {
```

Comment:
The class could be better tested using the MockDatabaseService.

Reply:
Yes, I think it can be refactored in a future separate brach.